test-framework/common/src/main/java/io/quarkus/test/common/TestHostLauncher.java

```diff
@@ -28,7 +28,7 @@ public void close() throws IOException {
 
     @Override
     public boolean listensOnSsl() {
-        return false;
+        return "true".equals(System.getProperty("quarkus.http.test-ssl-enabled"));
```

Comment:
```suggestion
        return Boolean.getBoolean("quarkus.http.test-ssl-enabled");
```

Reply:
Also I am not sure we should use that property name, since you'll get a warning when it's set (because it doesn't exist in any `@ConfigMapping/@ConfigRoot`)